### DISCUSSION TITLE
blog/%ED%86%A0%EB%A7%88%ED%86%A0%EB%93%A4-%ED%94%84%EB%A1%9C%EC%A0%9D%ED%8A%B8%EB%A5%BC-%EB%A7%BA%EC%9C%BC%EB%A9%B4%EC%84%9C-aa343ecd-f795-4aba-a11a-bb3e071df951

### DISCUSSION THREAD

**Discussion by giscus** [starcradle101/hoonie-log#54] [Comments]
# blog/%ED%86%A0%EB%A7%88%ED%86%A0%EB%93%A4-%ED%94%84%EB%A1%9C%EC%A0%9D%ED%8A%B8%EB%A5%BC-%EB%A7%BA%EC%9C%BC%EB%A9%B4%EC%84%9C-aa343ecd-f795-4aba-a11a-bb3e071df951

3주동안의 프로젝트를 돌아보며

https://hoonie-log.vercel.app/blog/%ED%86%A0%EB%A7%88%ED%86%A0%EB%93%A4-%ED%94%84%EB%A1%9C%EC%A0%9D%ED%8A%B8%EB%A5%BC-%EB%A7%BA%EC%9C%BC%EB%A9%B4%EC%84%9C-aa343ecd-f795-4aba-a11a-bb3e071df951

**Comment by hminah0215** [comment]
회고 작성도 아주 잘했네! 역시 👍
나도 참고해서 꼭 이번주안에 회고를 써봐야겠다.
